server/config.py

```diff
@@ -44,7 +44,7 @@
 CLIENT_STALL_TIME = int(os.getenv('CLIENT_STALL_TIME', 10))
 # Maximum number of bytes we will allow a stalled connection to get behind
 # before we terminate their connection.
-CLIENT_MAX_WRITE_BUFFER_SIZE = int(os.getenv('CLIENT_MAX_WRITE_BUFFER_SIZE', 2**16))
+CLIENT_MAX_WRITE_BUFFER_SIZE = int(os.getenv('CLIENT_MAX_WRITE_BUFFER_SIZE', 2**17))
```

Comment:
How was this related / why was it necessary?

Reply:
It was terminating more connections than I wanted it to and I realized that I probably should set the max buffer size higher than the default for the high watermark.